docker-compose.dev.yml

```diff
@@ -19,13 +22,20 @@ services:
       - ./_dev_volumes/private_documents:/private_documents
       - ./_dev_volumes/archive:/archive
     depends_on:
-      - "postgres"
+      postgres:
+        condition: service_healthy
     ports:
       - "${DJANGO_DOCKER_PORT}:9000"
 
   postgres:
-    image: postgis/postgis:13-3.2
+    image: postgis/postgis:16-3.4
```

Comment:
⚠️ This will, in terme, required from production instances to upgrade postgres servers ⚠️ First check that is is fine for instances admins. That said, I like the idea.

Reply:
Restored back to main.